docs/references/strategies/languages/gradle/gradle.md

```diff
@@ -282,6 +283,58 @@ We classify following configurations, and dependencies originating from it as a
 - kotlinNativeCompilerPluginClasspath
 ```
 
+## Only analyzing specific sub project
+
+If you have gradle project which has one or more subprojects, you may
+only want to analyze a specific set of subprojects in some cases.
+
+
+In `fossa-cli`, this can be achieved by using [exclusion filtering](./../../../files/fossa-yml.md).
+
+1) Run `fossa list-targets`, to identify project directory and identifier of subprojects.
+```bash
+[ INFO] Found project: gradle@./
+[ INFO] Found target: gradle@./::app
+[ INFO] Found target: gradle@./::list
+[ INFO] Found target: gradle@./::utilities
+```
+
+2) Now to analyze only `utilities`, use a `.fossa.yml` file in the project root.
+
+```yaml
+# filename: .fossa.yml
+#
+# analyze only gradle@./::utilities
+version: 3
+targets:
+  only:
+    - type: gradle
+      path: ./
+      target: ':utilities'
```

Comment:
Two colons in comment and in `list-targets` output, but one in the `target` setting- is this intentional? If so please explicitly call it out in the comment. Same with the other example below.

Reply:
yes, it is intentional; added: https://github.com/fossas/fossa-cli/pull/1059/commits/405cc0cee0dd5940663c9baa228dd86db1ab783e